### ISSUE TITLE
Notifications: cannot navigate into link via keybindings

### ISSUE THREAD

**Issue by tecton** [microsoft/vscode#163086]
Since notification message supports Markdown, we can add link in message and user can click the link using mouse. But user cannot focus on the link using keyboard, VS Code could support navigating with keyboard, the focus should reach to the link.

E.g. The "output window" link in message.
![image](https://user-images.githubusercontent.com/886116/194737299-496a1a4c-48de-4a83-ae93-87383fb6e2e2.png)

**Comment by bpasero** [comment]
Indeed. Fyi @meganrogge @isidorn

**Comment by bpasero** [comment]
The underlying reason is actually this:

https://github.com/microsoft/vscode/blob/96db0b339360bc72d27d869d81a557540ad5f800/src/vs/workbench/browser/parts/notifications/notificationsViewer.ts#L396

And changing it to `tabIndex: 0` allows to focus into the link. However, keybindings such as `Enter` are consumed by the notification itself to collapse/expand, so it is unfortunately not a 1-line change...

**Comment by VSCodeTriageBot** [comment]
This bug has been fixed in the latest release of [VS Code Insiders](https://code.visualstudio.com/insiders/)!

@tecton, you can help us out by commenting `/verified` if things are now working as expected.

If things still don't seem right, please ensure you're on version 1cf6a0823e31e26b55882af02afd41c650d461b8 of Insiders (today's or later - you can use `Help: About` in the command palette to check), and leave a comment letting us know what isn't working as expected.

Happy Coding!
